content/glossary/sql/alter-table.md

```diff
@@ -0,0 +1,37 @@
+<!--
+---
+Title: "ALTER TABLE"
+Subjects:
+  - "sql"
+  - "data science"
+Tags:
+  - "sql"
+  - "data science"
+  - "database"
+  - "queries"
+  - "table"
+  - "postgressql"
+  - "mysql"
+  - "sqlite"
+Catalog Content:
+  - "https://www.codecademy.com/learn/learn-sql"
+  - "https://www.codecademy.com/learn/sql-table-transformation"
+  - "https://www.codecademy.com/learn/paths/design-databases-with-postgresql"
+  - "https://www.codecademy.com/learn/paths/full-stack-engineer-career-path"
+---
+-->
+
+# ALTER TABLE
+
+## Definition 
+Adds, deletes or modifies or changes the data type of a column in a table.
+
+## Code Example
```

Comment:
@sonnynomnom I feel like we should show input and output? (well, pre-modification and post-modification tables)

Reply:
I'll add some tables in markdown to this file and let you guys see what it looks like on this next commit. I'll just do this ALTER TABLE file, for now, if you like the way it looks, I'll add it to other SQL entries in later commits.

Reply:
[Here](https://github.com/leftskidlow/codepedia-content/blob/main/content/glossary/sql/alter-table.md) is the ALTER TABLE with the tables included. Let me know what y'all think.